copy-this-template-directory/index.md

```diff
@@ -1,16 +1,924 @@
+    "calendar_event_end": "{{$trigger.body.data.calendar_event_end}}",
+    "name": "{{$trigger.body.data.name}}",
+    "description": "{{$trigger.body.data.description}}"
+}
+```
+
+
+***
+
+- Node 12 - "Transform Payload"
+
+![node 12](/copy-this-template-directory/directus_flow_4_12.png "node 12")
+
+if there was an error during operation. This node is not necessary
+
+
+After you save this Flow, copy the resulting webhook URL somewhere.
+
+***
+
+&nbsp; 
+
+
+## Setup in Google side
+
+[Google Apps Script](https://developers.google.com/apps-script/overview) is usually enabled by default, but if you are using Google Workspace within the organization, your Admin might disable Google Apps Script.
+
+Google Apps Script can be created as a dedicated file (Drive → New → More → Google Apps Script).
+
+But for this project, we will use Script within Spreadsheet. That way it is easier to use a spreadsheet for config and for logging.
+
+For quicker setup, please use [this spreadsheet template copy](https://docs.google.com/spreadsheets/d/1mXuSVR01uueeRDTjNeSXFH2rAziWkXdrNb2Jym413Fw/copy), it has full script inside and sheet with config.
+
+In your spreadsheet copy, open the menu Extensions → Apps Script. It will open Script Editor.
+
+![GAS 04](/copy-this-template-directory/GAS_04.png "GAS 04")
+
+Click the button `Deploy` → `New Deployment`.
+
+When Google Apps Script is deployed as Web App, it creates a unique URL like `https://script.google.com/macros/s/xxxx/exec`.
+When this URL is called with a GET request, the script function `doGet` is executed.  When this URL is called with a POST request, script function `doPost` is executed. We will use POST requests and doPost function.
+ 
+ ***
+
+![GAS 05](/copy-this-template-directory/GAS_05.png "GAS 05")
+
+set type to web app. Description - anything. Execute as - Me. Who has access - anyone.
+Click `Deploy`.
+
+ ***
+
+![GAS 01](/copy-this-template-directory/GAS_01.png "GAS 01")
+
+ click `Authorize access`
+ ***
+
+![GAS 02](/copy-this-template-directory/GAS_02.png "GAS 02")
+
+ 
+Choose your account and then click "Allow".
+
+Some users (usually, users not within the organization) will see a scary-looking
+
+"Google hasn’t verified this app
+The app is requesting access to sensitive info in your Google Account. Until the developer (your email) verifies this app with Google, you shouldn't use it."
+
+Click "Advanced"
+
+Then click "Go to code (unsafe)"
+
+ 
+ ***
+
+![GAS 03](/copy-this-template-directory/GAS_03.png "GAS 03")
+
+click "Allow".
+
+If all is fine, you'll see next popup with URL of your published web app.
+
+Web app
+
+URL
+
+`https://script.google.com/macros/s/xxxxxxxxxxxxxxxxxxx/exec`
+
+Copy
+
+click `copy` to get this Web app URL (make sure that you are copying web app URL and not Library URL) and update Directus Environment Variable `GCALENDARHOOKURL` to this value
+
+
+ ***
+
+ Back to the script editor
+
+![GAS 06](/copy-this-template-directory/GAS_06.png "GAS 06")
```

Comment:
Another example of why alt text is super important. 'In the action bar at the top of the script editor there is a run button and a dropdown of function names'